specs/unit/validation/place-order-request-validator.specs.ts

```diff
@@ -163,6 +163,47 @@ describe('Place Order Request Validator', () => {
             }
         });
     });
+    describe('Order Minimum Value Validation', () => {
+        describe('Collection delivery method', () => {
```

Comment:
The reason you can't add proper tests here is because of Cash payment method not being implemented. So feel free to delete any Collection tests for now.